Source/Tests/Trakt.NET.Modules.Tests/TraktUsersModule/TraktUsersModule_GetPersonalListsStream_Tests.cs

```diff
@@ -0,0 +1,167 @@
+
+        [Fact]
+        public async Task Test_TraktUsersModule_GetPersonalListStreamAsync()
+        {
+            TraktMultipleUserListsQueryParams parameters = new TraktMultipleUserListsQueryParams
+            {
+                { USERNAME, LIST_ID },
+                { USERNAME, LIST_ID },
+            };
+            int totalLists = parameters.Count;
+            TraktClient client = TestUtility.GetMockClientForMultipleCalls(GET_PERSONAL_LIST_STREAM_URI, LIST_JSON, totalLists);
+            IAsyncEnumerable<TraktResponse<ITraktList>> responses = client.Users.GetPersonalListsStreamAsync(parameters);
+
+            int returnedLists = 0;
+            await foreach (TraktResponse<ITraktList> response in responses.ConfigureAwait(false))
+            {
+                response.Should().NotBeNull();
+                response.IsSuccess.Should().BeTrue();
+                response.HasValue.Should().BeTrue();
+                response.Value.Should().NotBeNull();
+
+                ITraktList responseValue = response.Value;
+
+                responseValue.Name.Should().Be("Star Wars in machete order");
+                responseValue.Description.Should().Be("Next time you want to introduce someone to Star Wars for the first time, watch the films with them in this order: IV, V, II, III, VI.");
+                responseValue.Privacy.Should().Be(TraktAccessScope.Public);
+                responseValue.DisplayNumbers.Should().BeTrue();
+                responseValue.AllowComments.Should().BeFalse();
+                responseValue.SortBy.Should().Be("rank");
+                responseValue.SortHow.Should().Be("asc");
+                responseValue.CreatedAt.Should().Be(DateTime.Parse("2014-10-11T17:00:54.000Z").ToUniversalTime());
+                responseValue.UpdatedAt.Should().Be(DateTime.Parse("2014-11-09T17:00:54.000Z").ToUniversalTime());
+                responseValue.ItemCount.Should().Be(5);
+                responseValue.CommentCount.Should().Be(1);
+                responseValue.Likes.Should().Be(2);
+                responseValue.Ids.Should().NotBeNull();
+                responseValue.Ids.Trakt.Should().Be(55);
+                responseValue.Ids.Slug.Should().Be("star-wars-in-machete-order");
+                responseValue.User.Should().NotBeNull();
+                returnedLists++;
+            }
+            returnedLists.Should().Be(totalLists);
+        }
+
+        [Fact]
+        public async Task Test_TraktUsersModule_GetPersonalListStreamAsync_With_OAuth_Enforced()
+        {
+            TraktMultipleUserListsQueryParams parameters = new TraktMultipleUserListsQueryParams
+            {
+                { USERNAME, LIST_ID },
+                { USERNAME, LIST_ID },
+            };
+            int totalLists = parameters.Count;
+            TraktClient client = TestUtility.GetOAuthMockClient(GET_PERSONAL_LIST_STREAM_URI, LIST_JSON);
```

Comment:
The client is not setup correctly.
This should be `TestUtility.GetMockClientForMultipleCalls()`, but with OAuth.
You need to add an extra method similar to `TestUtility.GetMockClientForMultipleCalls()`, but with OAuth, like in `TestUtility.GetOAuthMockClient()`.